R/data-storage-log-file.R

```diff
@@ -70,6 +70,9 @@ DataStorageLogFile <- R6::R6Class( # nolint object_name_linter
         values, bucket = bucket, add_username = add_username, force_params = force_params
       )
 
+      if (!is.null(values$value))
+        values$value <- as.character(values$value)
+
```

Comment:
This should fix the issue with reading from the log file. Currently, the deployed API still uses the insert method that does not convert values to strings before logging, so the demo analytics app should still fail. It should be fixed once this branch is merge and the plumber API redeployed.